src/charts/helpers/text.js

```diff
@@ -149,11 +149,21 @@ define(function(require) {
         b.font = fontSize + 'px ' + fontFace;
 
         return b.measureText(text).width;
+    };
+
+    /**
+     * Gets the font size of the passed node using getComputedStyle
+     * @param {HTMLElement} node The node to get the computed font size for
+     * @return {number}
+     */
+    function getFontSize(node) {
+        return parseFloat(window.getComputedStyle(node).fontSize);
```

Comment:
Let's include here a check for the existence of 'window.getComputedStyle' (in Britecharts React we use Jest, that uses jsDOM and maybe doesn't have this method)

Reply:
Just FYI, there also appears to be an unchecked getComputedStyle call in src/charts/helpers/style.js, line 31